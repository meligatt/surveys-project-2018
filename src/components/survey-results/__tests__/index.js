import React from 'react';
import SurveyResults from '../index';

describe('SurveyResults', () => {
  it('renders correctly', () => {
    const wrapper = shallow(
      <SurveyResults match ={{params:{id: 1}}}/>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
