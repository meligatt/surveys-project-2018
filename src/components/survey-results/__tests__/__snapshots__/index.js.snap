// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SurveyResults renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <SurveyResults
    match={
        Object {
            "params": Object {
              "id": 1,
            },
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <SurveyDetails
          name={undefined}
          participantCount={undefined}
          responseRate={undefined}
          submittedResponseCount={undefined}
/>,
        <PercentagePie
          value={undefined}
/>,
        <AverageRatingPerQuestion
          themes={undefined}
/>,
        null,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "name": undefined,
          "participantCount": undefined,
          "responseRate": undefined,
          "submittedResponseCount": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "value": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "themes": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      null,
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <SurveyDetails
            name={undefined}
            participantCount={undefined}
            responseRate={undefined}
            submittedResponseCount={undefined}
/>,
          <PercentagePie
            value={undefined}
/>,
          <AverageRatingPerQuestion
            themes={undefined}
/>,
          null,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "name": undefined,
            "participantCount": undefined,
            "responseRate": undefined,
            "submittedResponseCount": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "value": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "themes": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        null,
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
